packages/common/src/schemas/json/lightdash-0.1.json

```diff
@@ -0,0 +1,190 @@
+                                                        "format": {
+                                                            "type": "string",
+                                                            "enum": [
+                                                                "km",
+                                                                "mi",
+                                                                "usd",
+                                                                "gbp",
+                                                                "eur",
+                                                                "percent"
+                                                            ]
+                                                        }
+                                                    },
+                                                    "required": ["type"]
+                                                }
+                                            }
+                                        },
+                                        "dimension": {
+                                            "type": "object",
+                                            "properties": {
+                                                "label": {
+                                                    "type": "string"
+                                                },
+                                                "description": {
+                                                    "type": "string"
+                                                },
+                                                "sql": {
+                                                    "type": "string"
+                                                },
+                                                "hidden": {
+                                                    "type": "boolean"
+                                                },
+                                                "round": {
+                                                    "type": "number",
+                                                    "minimum": 0
+                                                },
+                                                "format": {
+                                                    "type": "string",
+                                                    "enum": [
+                                                        "km",
+                                                        "mi",
+                                                        "usd",
+                                                        "gbp",
+                                                        "eur",
+                                                        "percent"
+                                                    ]
+                                                },
+                                                "time_intervals": {
```

Comment:
We are missing a few possible values here. You can check the docs on all possible values https://docs.lightdash.com/references/dimensions#dimension-configuration

Reply:
Thanks!